### ISSUE TITLE
Issue with training on the example set.

### ISSUE THREAD

**Issue by Hvsk22** [Nixtla/neuralforecast#590]
**Describe the bug**
I was trying to run the code given in one of the samples: https://nixtla.github.io/neuralforecast/examples/getting_started_complete.html
but received the error while training the AUTONHITS and AUTOLSTM models. 

**To Reproduce**
![image](https://github.com/Nixtla/neuralforecast/assets/42031978/4bf60949-61b5-4168-b3bf-4279a6c3dd43)
The rest of the code is exactly the same as in the tutorial. 

I believe this might have happened due to the changes that happened in the PyTorch-lightning update recently.

**Comment by kdgutier** [comment]
Hey @Hvsk22,

I think this bug, is due to a mismatch between the train loss and the validation loss.
Can you try to set them to match `loss=MQLoss(level=[80,90])` and `valid_loss=MQLoss(level=[80,90])`.

If this is the case can you confirm, I will add an Exception protection.

**Comment by Hvsk22** [comment]
Hey @kdgutier,

Thanks for the quick response.  I tried the above code but still had the same error. 
![image](https://github.com/Nixtla/neuralforecast/assets/42031978/0fd09893-24d8-4dbe-8129-32684627974c)

**Comment by kdgutier** [comment]
Thanks a lot for reporting the bug :beetle: @Hvsk22 

The fix will soon be in the main branch. You can install it with:
! pip install git+https://github.com/Nixtla/neuralforecast.git@main
